getJobs.js

```diff
@@ -0,0 +1,53 @@
+module.exports.getJobs = function getJobs(requestedGroupID) {
+  return new Promise((resolve) => {
+
+    const apiToken = process.env.API_TOKEN; 
```

Comment:
Constants should also be put in top level, or you can create a new file to put all server config called `config.js` and put all env var in it.